lms/data_tasks/report/create_from_scratch/02_entities/02_groups/01_raw_groups/01_create_view.jinja2.sql

```diff
@@ -22,12 +22,13 @@ CREATE VIEW report.raw_groups AS (
                 -- Every authority provided id should be unique to a type. But
                 -- we sometimes get blank values (in _very_ few cases), but to
                 -- be sure we should do a MAX here.
-                MAX("type") AS type
+                MAX("type") AS type,
+                MAX(lms_name) AS name
```

Comment:
This has kind of undefined behavior for conflicting info, but avoids nulls and is hopefully "Good enough (tm)"